src/modules/ResetPasswordConfirm/components/FormResetPasswordConfirm.jsx

```diff
@@ -0,0 +1,159 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Button, InputGroup, Form } from "react-bootstrap";
+
+import * as API from "api/http/users";
+import Input from "common/components/Input";
+
+export default class FormResetPasswordConfirm extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            formData: {
+                newPassword: "",
+                retypeNewPassword: "",
+            },
+
+            errorMsgs: [],
+            buttonState: false,
+            resetStatus: false,
+        };
+
+        this.fieldRefs = {
+            newpassword: React.createRef(),
+            retypeNewPassword: React.createRef(),
+        };
+
+        this.onChange = this.onChange.bind(this);
+        this.onSubmit = this.onSubmit.bind(this);
+        this.passwordValidation = this.passwordValidation.bind(this);
+    }
+
+    onChange(name, value) {
+        this.setState((prevState) => ({
+            formData: { ...prevState.formData, [name]: value },
+        }));
+
+        Object.keys(this.fieldRefs).forEach((ref) => {
+            if (ref !== name) {
+                this.fieldRefs[ref].current.forceUpdate();
+            }
+        });
+    }
+
+    onSubmit(event) {
+        event.preventDefault();
+        const { formData } = this.state;
+
+        const urlParams = new URLSearchParams(window.location.search);
+        const token = urlParams.get("token");
+
+        API.passwordResetConfirm(formData.newPassword, token)
+            .then((response) => {
+                if (response.status === 200) {
+                    this.setState(() => ({ resetStatus: true }));
+                }
+            })
+            .catch((error) => {
+                if (error.response.status === 400) {
+                    this.setState(() => ({ errorMsgs: error.response.data.password }));
+                }
+            });
+    }
+
+    passwordValidation() {
+        const { formData } = this.state;
+        const isValid = formData.newPassword === formData.retypeNewPassword;
+
+        this.setState(() => ({
+            buttonState: !isValid,
+        }));
+
+        return isValid;
+    }
+
+    render() {
+        const {
+            formData, buttonState, resetStatus, errorMsgs,
+        } = this.state;
+
+        return (
+            <div>
+                {resetStatus ? [
+                    <p key="title" className="login-box-msg">Reset password</p>,
+                    <p key="text">Password was successfully changed</p>,
+                    <a
+                        key="to-sign-in-link"
+                        className="btn btn-primary btn-block"
+                        href="/signin"
+                    >
+                        Return to sign in
+                    </a>,
+                ] : [
+                    <p key="title" className="login-box-msg">
+                        You are only one step a way from your new password,
```

Comment:
1) there's a typo;
2) the line doesn't provide any important information.

Let's get rid of this.